eta/core/types.py

```diff
@@ -489,6 +489,41 @@ def is_valid_path(path):
         )
 
 
+class NPZFile(File, ConcreteData):
```

Comment:
I don't think these are needed anymore as you updated the tracking modules to use directories rather than sequences of files like `/path/to/features/%05d.npz`. Can you please confirm?

Reply:
The Hungarian Matrix is stored as an `NPZFileSequence`.

Reply:
Oh I see. That's fine then.